### ISSUE TITLE
Setup Server

### ISSUE THREAD

**Issue by cfree** [denvercodeclub/denvercodeclub-com#7]
- [x] set up `package.json`
- [ ] set up server in `app.js` - get the server up and running using Express

Create this repo structure:

```
- public/
- public/app/
- public/assets/
- server/
```

**Comment by winnemucca** [comment]
Going to build this from scratch?

**Comment by cfree** [comment]
Not a machine, just using node.js

**Comment by cfree** [comment]
@winnemucca hah, late night Githubbing.. Yes, we'll build it from scratch for practice and forego using a Yeoman generator

**Comment by cfree** [comment]
I can't assign two people, but I want @ab7 to help out here too. Pair programming?

**Comment by cfree** [comment]
(Don't start yet, this is contingent on another task)

**Comment by ab7** [comment]
Count me in.

**Comment by cfree** [comment]
@winnemucca and @ab7 - you're good to start. We just need to get the server up, don't worry about routes, etc. right now.

Please make a separate branch and then create a pull request for review

**Comment by winnemucca** [comment]
Nice I will look at this saturday.
Thanks!

**Comment by cfree** [comment]
We're about 15 days behind schedule :) How are things looking?

**Comment by ab7** [comment]
I remember @winnemucca mentioning that he did some work on this. I don't want to redo what he has already done.

**Comment by winnemucca** [comment]
Hey, I did some very basic things for set up.  Wasn’t sure who had other jobs so I did not want to get into routing or setting up the models.  I created the app.js which took 5 minutes.  I believe that and the some structuring was the majority of my task.   I just pushed up what I did from a while ago.  Let me know if you got it.  
Steve
